### ISSUE TITLE
[IDEA]

### ISSUE THREAD

**Issue by KadTheHunter** [KadTheHunter/CityPlans#3]
**What is your idea? Please describe.**
Street name layout, adapting current [streets](https://github.com/KadTheHunter/CityPlans/blob/master/Streets) to fit the new world.

**Additional context**
N/A

Thinking that the [Kings Street](https://github.com/KadTheHunter/CityPlans/blob/master/Streets#L21) should be renamed to Kings Road and should be the road we're starting to angle (See **Signs** at _161 39 -60_, **Diagrams** at _136 40 -125_
)

[Wall Street](https://github.com/KadTheHunter/CityPlans/blob/master/Streets#L27) should be near City spawn, perhaps the road directly behind you when you warp there.

[Orion Boulevard](https://github.com/KadTheHunter/CityPlans/blob/master/Streets#L1) will stay the same, however, Trafalgar Square should be moved to the North end, and the cross streets (1 through 15) should be going south in ascending order (with 15 being the south-most and also the seediest), with [Wall Street](https://github.com/KadTheHunter/CityPlans/blob/master/Streets#L27) being the first or second street going north.

**Comment by KadTheHunter** [comment]
Noting here that the world layout has changed yet again, due to the newest changes this issue will be put on hold until we have much more built.

**Comment by ParadiseMCPE** [comment]
I'd be interested to hear what the topography of the area you are using is (or if that's even planned at all). If you have bits of the city separated by water like New York or very hilly areas like San Francisco. I think working the natural parts of the world into the way the roads look or are placed would be really cool.

**Comment by KadTheHunter** [comment]
> I'd be interested to hear what the topography of the area you are using is (or if that's even planned at all). If you have bits of the city separated by water like New York or very hilly areas like San Francisco. I think working the natural parts of the world into the way the roads look or are placed would be really cool.

Currently there isn't much in the way of topography. We've been winging it thus far, and so only have a couple streets and their buildings done. We do have a bridge, but it's not going above water, it's going above another boulevard because we totally screwed up when doing the math for the intersection with the angled road across the way :)

**Comment by ParadiseMCPE** [comment]
ah alright. depending on what server host you're using maybe I can help draw up a landscape with some world-edit.

**Comment by KadTheHunter** [comment]
> ah alright. depending on what server host you're using maybe I can help draw up a landscape with some world-edit.

Mmm, perhaps, but currently most landscaping will be done
- By hand
- Via pasted in Schematics
as WorldEdit (or, as we have, World**Styler**, which is lightyears faster than W.E.) is very clunky for making land, as is BlockSniper. The best way to make land is via a Java Minecraft application that creates realistic terrain on a massive scale, but this is impractical for us, as importing worlds to PocketMine that were made using that program is very very annoying, when possible.

**Comment by ParadiseMCPE** [comment]
Yeah if I had a PC I'd surely be using the Java Minecraft one, but as I'm on Pocket all I've ever known is LEET's. It'll take quite some time with the small team you've got right now but I trust in time you'll find a shortcut.
